.idea/workspace.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file?

Reply:
No clue, think it was added automatically